devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy

```diff
@@ -67,27 +68,35 @@ def updated() {
 	updateDataValue("onOff", "catchall")
 }
 
-// Parse incoming device messages to generate events
 def parse(String description) {
 	log.debug "description is $description"
-	Map map = zigbee.getEvent(description)
-	if (map) {
-		if (description?.startsWith('on/off')) {
-			log.debug "receive on/off message without endpoint id"
-			sendHubCommand(refresh().collect { new physicalgraph.device.HubAction(it) }, 0)
+	Map eventMap = zigbee.getEvent(description)
+	log.debug "eventMap is $eventMap"
+	Map eventDescMap = zigbee.parseDescriptionAsMap(description)
+	log.debug "eventDescMap is $eventDescMap"
+
+	if (!eventMap && eventDescMap) {
+		log.debug "create new eventMap "
+		eventMap = [:]
+		if (eventDescMap?.cluster == "0006" || eventDescMap?.clusterId == "0006") {
```

Comment:
use 

`eventDescMap?.clusterInt = zigbee.ON_OFF_CLUSTER`

so we can do an int comparison and get rid of magic numbers. Can also then remove the redundant conditional

Reply:
also is there any reason `getEvent` isn't parsing this fully to a switch event already?

Reply:
I noticed "eventDescMap?.cluster" while i was trying to deal with setLevel for this device some time ago, so that conditional was just in case only.
It will be changed to zigbee.ON_OFF_CLUSTER of course.